samples/openapi3/client/petstore/python/petstore_api/rest.py

```diff
@@ -146,6 +146,7 @@ def request(
                 elif headers['Content-Type'] == 'application/x-www-form-urlencoded':  # noqa: E501
                     r = self.pool_manager.request(
                         method, url,
+                        body=body,
                         fields=fields,
```

Comment:
Are fields unused for this content-type?
If so, how about removing that input from the request call.

Reply:
It looks like fields are unused per https://stackoverflow.com/questions/62392885/how-to-use-urllib3-to-post-x-www-form-urlencoded-data

Reply:
I'd be happy to this issue some other way, but this seemed like the minimal change to get x-www-form-urlencoded data working.